quartz/plugins/emitters/assets.ts

```diff
@@ -30,9 +35,8 @@ export const Assets: QuartzEmitterPlugin = () => {
       return graph
     },
     async emit({ argv, cfg }, _content, _resources): Promise<FilePath[]> {
-      // glob all non MD/MDX/HTML files in content folder and copy it over
       const assetsPath = argv.output
-      const fps = await glob("**", argv.directory, ["**/*.md", ...cfg.configuration.ignorePatterns])
+      const fps = await filesToCopy(argv, cfg)
```

Comment:
this doesn't seem to change the copying logic, are you missing a diff?

Reply:
The copying logic here is the same. I changed build.ts not to call this emitter unless an asset file is changed